rskj-core/src/test/java/co/rsk/peg/BridgeSupportAddSignatureTest.java

```diff
@@ -283,31 +283,31 @@ public void addSignatureFromInvalidFederator() throws Exception {
     }
 
     @Test
-    public void addSignatureWithInvalidSignature() throws Exception {
+    void addSignatureWithInvalidSignature() throws Exception {
         List<BtcECKey> list = new ArrayList<>();
         list.add(new BtcECKey());
         addSignatureFromValidFederator(list, 1, true, false, "InvalidParameters");
     }
 
     @Test
-    public void addSignatureWithLessSignaturesThanExpected() throws Exception {
+    void addSignatureWithLessSignaturesThanExpected() throws Exception {
         List<BtcECKey> keys = Collections.singletonList(BridgeRegTestConstants.REGTEST_FEDERATION_PRIVATE_KEYS.get(0));
         addSignatureFromValidFederator(keys, 0, true, false, "InvalidParameters");
     }
 
     @Test
-    public void addSignatureWithMoreSignaturesThanExpected() throws Exception {
+    void addSignatureWithMoreSignaturesThanExpected() throws Exception {
         List<BtcECKey> keys = Collections.singletonList(BridgeRegTestConstants.REGTEST_FEDERATION_PRIVATE_KEYS.get(0));
         addSignatureFromValidFederator(keys, 2, true, false, "InvalidParameters");
     }
 
     @Test
-    public void addSignatureNonCanonicalSignature() throws Exception {
+    void addSignatureNonCanonicalSignature() throws Exception {
         List<BtcECKey> keys = Collections.singletonList(BridgeRegTestConstants.REGTEST_FEDERATION_PRIVATE_KEYS.get(0));
         addSignatureFromValidFederator(keys, 1, false, false, "InvalidParameters");
     }
 
-    private void test_addSignature_EventEmitted(boolean rskip326Active, boolean useValidSignature, int wantedNumberOfInvocations, boolean shouldSignTwice) throws Exception {
```

Comment:
Should this one be removed? I believe it makes it public by default

Reply:
Done!